src/Microsoft.VisualStudio.Services.Agent/VstsAgentWebProxy.cs

```diff
@@ -140,7 +140,7 @@ private void LoadProxySetting()
 
             if (!string.IsNullOrEmpty(ProxyAddress) && !Uri.IsWellFormedUriString(ProxyAddress, UriKind.Absolute))
             {
-                Trace.Info($"The proxy url is not a well formed absolute uri string: {ProxyAddress}.");
+                Trace.Error($"The proxy url is not a well formed absolute uri string: {ProxyAddress}.");
```

Comment:
i think another option for improving the proxyurl error detection would be adding input validation for `--proxyurl` input.

Reply:
I agree, but this isn't passed in via --proxyurl, this is being read in via .proxy file (in the case that prompted the change). In order to do better validation, a lot more code would need to be restructured. Also, for backwards compatibility, we would need the agent to keep executing if the proxy url was malformed.

Reply:
However, whenever we take input from the user and ignore it, it should be marked as an error, not info.

Reply:
the documented way of config proxy is using `--proxy`, i think we should optimize for that.
still keep what you had with diag log error, but adding input validation as additional enhancement, so user can get early validation error.

Reply:
It looks like we already do have validation if they use --proxy